scripts/release.bat

```diff
@@ -32,11 +32,9 @@ set VERSION=%2
 IF "%VERSION%"=="2015" (
     set "DLLS=C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\redist\x86\Microsoft.VC140.CRT\msvc*.dll"
 ) ELSE (
-    IF EXIST "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Redist\MSVC\14.14.26405\x86\Microsoft.VC141.CRT\" (
-        set "DLLS=C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Redist\MSVC\14.14.26405\x86\Microsoft.VC141.CRT\msvc*.dll"
-		) ELSE (
-        set "DLLS=C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Redist\MSVC\14.14.26405\x86\Microsoft.VC141.CRT\msvc*.dll"
-    )
+    set "DLLS=MSVC_DLLS_NOT_FOUND"
+    FOR /d %%d IN ("C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Redist\MSVC\*"
+                   "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Redist\MSVC\*") DO set "DLLS=%%d\x86\Microsoft.VC141.CRT\msvc*.dll"
```

Comment:
Would it make sense to also use `*` for VC141 or is that fixed relative to msvc2017?

Reply:
My guess is that's fixed relative to msvc2017, but I'm not sure. Another wildcard would need another nested ``FOR``, though - in fact, originally I had a wildcard there as well, but then noticed that this wildcard construct only seems to work for a single ``*`` at the very end of a filename in batch scripts... that's why I removed it, since I reckon it will stay the same for msvc2017 anyways.